lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -221,6 +222,8 @@ private QuestionsResponse getQuestionAnswerResponse(KBVItem kbvItem)
             PersonIdentityDetailed personIdentity =
                     personIdentityService.getPersonIdentityDetailed(kbvItem.getSessionId());
             QuestionRequest questionRequest = new QuestionRequest();
+            questionRequest.setStrategy(
+                    this.configurationService.getParameterValue(IIQ_STRATEGY_PARAM_NAME));
```

Comment:
@techaman-m could you add a verify to the test for this class that checks that the `ConfigurationService` gets the IIQStrategy from the `getParameterValue` method please?

Reply:
Added `verify` in a few tests